### PR TITLE
Update Vale rules

### PR DESCRIPTION
**Pull request by andreyaksenov**
- Added a suggestion for swapping `flag` with `option` in a case of CLI options.
- Made some rules a bit weaker.
- Added missing terms.

### CODE REVIEW

File: styles/Vocab/Tarantool/accept.txt [hunk 1]
```diff
@@ -33,4 +33,15 @@ storages
...
+autoincrement
+etcd
+xlog
+canonicalization
+libcurl
+CMake
+fuzzer
+subcommand
+vshard
+boolean
```
Comment (p7nov) [inline styles/Vocab/Tarantool/accept.txt]:
```suggestion
boolean
failover
```

File: styles/Vocab/Tarantool/accept.txt [hunk 2]
```diff
@@ -33,4 +33,15 @@ storages
 dereference
 upsert
 tt
-SEQSCAN
\ No newline at end of file
+SEQSCAN
+iproto
+autoincrement
+etcd
+xlog
```
Comment (p7nov) [inline styles/Vocab/Tarantool/accept.txt]: Not sure we should encourage using "xlog" as a word in plain text. The correct entity name is "write-ahead log".

File: styles/Tarantool/Uppercase.yml [hunk 3]
```diff
@@ -251,3 +251,5 @@ exceptions:
   - YAML
   - ZAP
   - ZIP
+  - PAP
+  - CHAP
```
Comment (p7nov) [inline styles/Tarantool/Uppercase.yml:255]:
```suggestion
  - CHAP
  - APT
```

**Review by p7nov — APPROVED** [review]
Thanks for this update!
Added a couple more suggestions.

**Review by xuniq — APPROVED** [review]

**Pull request merged**